src/index.js

```diff
@@ -0,0 +1,58 @@
+import { createModal } from "./modal.js";
+
+const dropDownClickable = document.querySelector(".dropDown_clickable");
+const dropDownSelected = document.querySelector(".dropDown_clickable span");
+const dropDownContent = document.querySelector(".dropDown_content");
+const dropDownContents = document.querySelectorAll(".dropDown_content a");
+
+let state = "off";
+function handleClikableClick() {
+  if (state === "off") {
+    dropDownContent.style.display = "flex";
+    state = "on";
+  } else {
+    dropDownContent.style.display = "none";
+    state = "off";
+  }
+}
+
+function handleMouseenter(event) {
+  let selected = event.currentTarget;
+  selected.style.backgroundColor = "lightgrey";
+}
+
+function handleMouseleave(event) {
+  let selected = event.currentTarget;
+  selected.style.backgroundColor = "whitesmoke";
+}
```

Comment:
이 부분은 클래스로 스타일을 준다면 코드를 간소화시킬 수 있을 것 같아욤
```suggestion
function handleMouseEnterLeave(event, isEntering) {
  let selected = event.currentTarget;
  selected.classList.toggle("hovered", isEntering); // 클래스로 스타일 변경
}

dropDownContents.forEach((content) => {
  content.addEventListener("mouseenter", (e) => handleMouseEnterLeave(e, true));
  content.addEventListener("mouseleave", (e) => handleMouseEnterLeave(e, false));
  content.addEventListener("click", handleContentClick);
});
```

Reply:
이처럼 수정한다면 이벤트 리스너가 중복되는 부분도 해결할 수 있을 것 같아요 : )

Reply:
@Nahee-Park 이렇게 수정하려고 했는데 CSS 적용이 잘 안되네요 ㅠㅠ isEntering이 될 때마다 해당 html 태그 class에 hovered 가 정상적으로 추가되는 것도 확인했는데... 왜 적용이 안될까요?
해당 커밋 내역입니다! => [#](https://github.com/GDSC-Ewha-5th/leeNaHyun/pull/2/commits/a4424f7d9a6a8b1264f8f6f9b3d981bda3659257)